scripts/rust-analyze.sh

```diff
@@ -35,6 +35,8 @@ PLATFORM_SAVE_FILTER=${PLATFORM:+${RUST_PLAT_DIRS[$PLATFORM]}/debug/deps/}
 PLATFORM_FILTER=${PLATFORM:+${RUST_PLAT_DIRS[$PLATFORM]}/}
 PLATFORM_SUFFIX=${PLATFORM:+-$PLATFORM}
 
+SED_TRANSFORM_EXPR=$4
```

Comment:
Won't this trigger an error if `$4` isn't defined?

Reply:
It did in fact trigger an error!